src/_includes/common-form-fields.html

```diff
@@ -0,0 +1,134 @@
+{# ==========================================================================
+
+   common_form_fields.state()
+
+   ==========================================================================
+
+   Description:
+
+   Returns a select box with all 50 states and DC.
+
+   legend: Name of the field.
+
+   size: The size of the fields. Defaults to 1-3.
+          Possible options: 1, 1-2, 1-3
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro state(size='1-3', required=false, disabled=false) %}
+  {% from 'macros/form.html' import select as select %}
+  {{ select('State',
+            'Select state',
+            [ 'Alabama',
+              'Alaska',
+              'Arizona',
+              'Arkansas',
+              'California',
+              'Colorado',
+              'Connecticut',
+              'Delaware',
+              'District of Columbia',
+              'Florida',
+              'Georgia',
+              'Hawaii',
+              'Idaho',
+              'Illinois',
+              'Indiana',
+              'Iowa',
+              'Kansas',
+              'Kentucky',
+              'Louisiana',
+              'Maine',
+              'Montana',
+              'Nebraska',
+              'Nevada',
+              'New Hampshire',
+              'New Jersey',
+              'New Mexico',
+              'New York',
+              'North Carolina',
+              'North Dakota',
+              'Ohio',
+              'Oklahoma',
+              'Oregon',
+              'Maryland',
```

Comment:
Ah! I started reading from MD->MO and couldn't find which one was out of order in that block!